ReactTask/src/views/DoctorProfile.js

```diff
@@ -45,10 +46,22 @@ const DoctorProfile = ({ title }) => {
           </StylBoxDoctor>
         </StylBoxRowPadding>
         {!!dataProfileDoctor.length && (
-          <CardPatientProfile
-            dataPatients={dataProfileDoctor}
-            imgIconDescription={scheduleImgSvg}
-          />
+          <StylBoxPatientsList>
+            {dataProfileDoctor.map(
+              ({ id, alt, name, status, statusText, data, info }) => (
+                <CardPatientProfile
+                key={id}
+                  imgIconDescription={scheduleImgSvg}
+                  alt={alt}
+                  name={name}
+                  status={status}
+                  statusText={statusText}
+                  data={data}
+                  info={info}
```

Comment:
Why dont you import appointent into the component, and make Destructuring in inner component?